app/services/jira_service.py

```diff
@@ -72,11 +72,17 @@ def get_project_issues(self, project_key):
         Returns a list of issue objects for a given project
         """
         if self._init_if_needed():
-            return self.client.search_issues(
-                'project = "' + project_key +
-                '" AND issuetype not in subtaskIssueTypes() AND' +
-                ' resolution = Unresolved'
-            )
+            try:
```

Comment:
We could probably remove the various `try/except` blocks now around various actions. They were the first attempt at a fix that allowed me to identify archived projects as the underlying issue.